parser-typechecker/src/Unison/Runtime/Builtin.hs

```diff
@@ -1775,63 +1800,63 @@ declareForeigns = do
   let updateClient :: X.CertificateStore -> TLS.ClientParams -> TLS.ClientParams
       updateClient certs client = client { TLS.clientShared = ((clientShared client) { TLS.sharedCAStore = certs }) } in
 
-        declareForeign "Tls.ClientConfig.certificates.set" boxBoxDirect . mkForeign $
+        declareForeign Tracked "Tls.ClientConfig.certificates.set" boxBoxDirect . mkForeign $
           \(certs :: [X.SignedCertificate], params :: ClientParams) -> pure $ updateClient (X.makeCertificateStore certs) params
 
   let updateServer :: X.CertificateStore -> TLS.ServerParams -> TLS.ServerParams
       updateServer certs client = client { TLS.serverShared = ((serverShared client) { TLS.sharedCAStore = certs }) } in
-        declareForeign "Tls.ServerConfig.certificates.set" boxBoxDirect . mkForeign $
+        declareForeign Tracked "Tls.ServerConfig.certificates.set" boxBoxDirect . mkForeign $
           \(certs :: [X.SignedCertificate], params :: ServerParams) -> pure $ updateServer (X.makeCertificateStore certs) params
 
-  declareForeign "TVar.new" boxDirect . mkForeign
+  declareForeign Untracked "TVar.new" boxDirect . mkForeign
     $ \(c :: Closure) -> unsafeSTMToIO $ STM.newTVar c
 
-  declareForeign "TVar.read" boxDirect . mkForeign
+  declareForeign Tracked "TVar.read" boxDirect . mkForeign
     $ \(v :: STM.TVar Closure) -> unsafeSTMToIO $ STM.readTVar v
 
-  declareForeign "TVar.write" boxBoxTo0 . mkForeign
+  declareForeign Tracked "TVar.write" boxBoxTo0 . mkForeign
     $ \(v :: STM.TVar Closure, c :: Closure)
         -> unsafeSTMToIO $ STM.writeTVar v c
 
-  declareForeign "TVar.newIO" boxDirect . mkForeign
+  declareForeign Untracked "TVar.newIO" boxDirect . mkForeign
     $ \(c :: Closure) -> STM.newTVarIO c
 
-  declareForeign "TVar.readIO" boxDirect . mkForeign
+  declareForeign Tracked "TVar.readIO" boxDirect . mkForeign
     $ \(v :: STM.TVar Closure) -> STM.readTVarIO v
 
-  declareForeign "TVar.swap" boxBoxDirect . mkForeign
+  declareForeign Untracked "TVar.swap" boxBoxDirect . mkForeign
     $ \(v, c :: Closure) -> unsafeSTMToIO $ STM.swapTVar v c
 
-  declareForeign "STM.retry" unitDirect . mkForeign
+  declareForeign Untracked "STM.retry" unitDirect . mkForeign
     $ \() -> unsafeSTMToIO STM.retry :: IO Closure
 
   -- Scope and Ref stuff
-  declareForeign "Scope.ref" boxDirect
+  declareForeign Untracked "Scope.ref" boxDirect
     . mkForeign $ \(c :: Closure) -> newIORef c
 
-  declareForeign "IO.ref" boxDirect
+  declareForeign Untracked "IO.ref" boxDirect
```

Comment:
I guess it's okay (if somewhat pointless) to have these `Untracked`.

Reply:
I decided to make these tracked anyway, just for uniformity with making pretty much all other mutable stuff tracked.